copper-server-api/src/main/java/com/pogeyan/cmis/api/data/ISpan.java

```diff
@@ -0,0 +1,16 @@
+package com.pogeyan.cmis.api.data;
+
+import java.util.Map;
+
+public interface ISpan {
+
+	void setChildSpan(String baseMessageId, String name, Map<String, String> headers);
```

Comment:
rename from "baseMessageId" to "tracingId", since we are considering baseMessageId as tracing.